k8s/chart/README.md

```diff
@@ -0,0 +1,69 @@
+# SDP Operator Helm Chart
+
+## Quickstart
+```
+helm install sdp-operator ghcr.io/appgate/sdp-operator:latest
```

Comment:
Once github workflow is created, this needs to change to `helm install sdp-operator ghcr.io/appgate/charts/sdp-operator --version <version>`